src/Postmark/PostmarkClient.cs

```diff
@@ -858,31 +858,34 @@ public async Task<PostmarkResponse> SendEmailWithTemplateAsync(TemplatedPostmark
 
         public async Task<PostmarkResponse> SendEmailWithTemplateAsync<T>(string templateAlias, T templateModel,
             string to, string from,
+            string messageStream = null,
```

Comment:
Due to the `Attachment` `params` at the end, we cannot add this argument at the end of the function signature, so we must break backwards compatibility. This probably requires a major version update.

I could also combine this major version update with the `BounceID` long conversion.